containers/anndata-to-ui/VERSION

```diff
@@ -1 +1 @@
-0.0.6
+0.0.5
```

Comment:
I changed this to match the docker-image version specified in the `.cwl` file

Reply:
We do have a v0.0.6 that's pushed in the docker tags, so we might need to make the adjustment in the cwl file instead: https://hub.docker.com/r/hubmap/portal-container-anndata-to-ui/tags